app/views/answerbox/_smiles.haml

```diff
@@ -7,6 +7,6 @@
     - else
       - a.smiles.all.each do |smile|
         %a{ href: show_user_profile_path(smile.user.screen_name),
-          title: smile.user.screen_name,
+          title: smile.user.display_name,
```

Comment:
What if the display name is empty tho?

Use the designated helper `UserHelper#user_screen_name` for this: 
https://github.com/Retrospring/retrospring/blob/master/app/helpers/user_helper.rb

```suggestion
          title: user_screen_name(smile.user, anonymous: false, url: false),
```